Project1/src/scripts/S4.java

```diff
@@ -0,0 +1,8 @@
+package scripts;
+
+public class S4 
+{
+	/*
+	 * added after creating branch in eclipse
+	 */

```

Comment:
add logic & send

Reply:
added logic. please approve

Reply:
code is reviewed and you can merge to master